app/assets/stylesheets/notifications.scss

```diff
@@ -85,6 +85,7 @@
             p.badge-reward-message {
               margin: 8px auto;
               width: 90%;
+              color: #000;
```

Comment:
I don't think that this will work with the different themes, here are the DEV theming guidelines: https://docs.dev.to/design/theming/

There are 2 options to add support for themes, I would advice to use the last one.

Reply:
Thank you @Glennmen for your advice to look into the theming. I updated so now the color changes based on which theme a user is using.